src/Uno.UI.Runtime.Skia.AppleUIKit/UI/Xaml/RootViewController.cs

```diff
@@ -195,7 +195,7 @@ private void ClipBySvgPath(string svg)
 							y = (y / scale - vy);
 							x2 = (x2 / scale - vx);
 							y2 = (y2 / scale - vy);
-							cgPath.AddQuadCurveToPoint(default, x, y, x2, y2);
+							cgPath.AddQuadCurveToPoint(x, y, x2, y2);
```

Comment:
This should not change anything at runtime. The MS code simply provides `null` for this overload (if not supplied) and there's only a single native call `CGPathAddQuadCurveToPoint`.

Reply:
Surprisingly it does! With this overload each of the curves started in the top left corner and created the visual glitches. The overload without this parameter just works as expected. Not sure why there is a difference, but the result looks right

Reply:
ah! I see what's wrong 😄 

`CGAffineTransform` is a `struct` and `default` would make all fields empty (all zero) but supplying `null` means to use an identity matrix.

That explains why this ObjC version of the code did not show the same problem.